frame/nomination-pools/src/migration.rs

```diff
@@ -449,3 +451,79 @@ pub mod v3 {
 		}
 	}
 }
+
+pub mod v4 {
+	use super::*;
+
+	#[derive(Decode)]
+	pub struct OldBondedPoolInner<T: Config> {
+		pub points: BalanceOf<T>,
```

Comment:
please check the order of these fields -- they must match the old ones.

Reply:
I think this is good but I don't think `v1` is valid - I added the `commission` field in the v1 migration to keep the compiler happy, but obviously this is not good for the v2 - v3 migrations. I've added a note, I think a new `struct` is needed there.

Reply:
Fields do indeed match the old ones